src/frontend/komponenter/Fagsak/Personlinje/Personlinje.tsx

```diff
@@ -65,4 +72,11 @@ const Personlinje: React.FC<IProps> = ({ bruker, minimalFagsak }) => {
     );
 };
 
+const sjekkOmMigreringsdatoErEldreEnn3År = (migreringsdatoIString: string) => {
+    const dato = new Date();
+    const migreringsdato = new Date(migreringsdatoIString);
+    const difference = dato.getTime() - migreringsdato.getTime();
+    return Math.floor(Math.abs(difference) / (1000 * 60 * 60 * 24 * 365));
```

Comment:
`1000 * 60 * 60 * 24 * 365` burde dette inn i en konstant? for eksempel MILLISEKUNDER_I_ÅRET.